--- /etc/dhcp/dhcpd.conf.orig     2016-03-09 17:23:10.299000000 +0300
+++ /etc/dhcp/dhcpd.conf  2016-03-09 17:24:29.759000000 +0300
@@ -3,3 +3,10 @@
 #   see /usr/share/doc/dhcp*/dhcpd.conf.example
 #   see dhcpd.conf(5) man page
 #
+ldap-server "localhost";
+ldap-port 389;
+ldap-username "cn=odmin,dc=lan";
+ldap-password "secred";
+ldap-base-dn "cn=server.lan,ou=DHCP,dc=lan";
+ldap-method static;
+authoritative;
