--- my.cnf.orig	2010-02-21 05:37:27.000000000 +0300
+++ my.cnf	2010-04-05 14:43:53.000000000 +0400
@@ -1,4 +1,9 @@
 [mysqld]
+default-character-set=utf8
+default-collation=utf8_general_ci
+character-set-server=utf8
+collation-server=utf8_general_ci
+init-connect='SET NAMES utf8'
 datadir=/var/lib/mysql
 socket=/var/lib/mysql/mysql.sock
 user=mysql
@@ -28,3 +33,6 @@
 [ndb_mgm]
 # connection string for MySQL Cluster management tool
 connect-string="host=localhost:1186"
+
+[client]
+default-character-set=utf8
