--- smbldap_bind.conf.orig	2016-03-12 13:09:37.292891200 +0300
+++ smbldap_bind.conf	2016-03-12 13:10:37.163898338 +0300
@@ -7,7 +7,7 @@
 # master ldap for writing access and a slave ldap server for reading access
 # By default, we will use the same DN (so it will work for standard Samba
 # release)
-slaveDN="cn=Manager,dc=example,dc=com"
-slavePw="secret"
-masterDN="cn=Manager,dc=example,dc=com"
-masterPw="secret"
+slaveDN="cn=odmin,dc=lan"
+slavePw="secred"
+masterDN="cn=odmin,dc=lan"
+masterPw="secred"
